### PR TITLE
Added new startup scripts that don't blast existing ISIS environment …

### PR DESCRIPTION
**Pull request by scsides**
New startup scripts that are not destructive to an existing environment

## Description

Added new startup scripts that don't overwrite the existing ISIS environment variables. If the variable is already set, the new scripts don't change it.

## Motivation and Context

Developing outside USGS network using the existing scripts is painful. These can be used instead of the ISIS3Startup.XXXX versions to work better.

## How Has This Been Tested?

The scripts were run locally, and the environment was examined to see that only unset variables were modified.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X} I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/scripts/isisStartup.csh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# > setenv ISISROOT ????
+# > source isis3Startup.csh
+#
+# Replace the "????" in the above command line with the path you installed
+# the Isis distribution
+#
+################################################################################
+
+if ($?ISISROOT == 0) then
+  setenv ISISROOT /usgs/pkgs/isis3/install
```
Comment (Kelvinrr) [inline isis/scripts/isisStartup.csh]: Do we still use this area? Should it use an anaconda package instead?

**Review by Kelvinrr — COMMENTED** [review]
just a (probably dumb) question.

**Comment by kaitlyndlee** [comment]
I see the differences between the these scripts and the ones that are currently in the scripts directory, but why do we need two separate pairs of scripts with very similar names? I feel like this could be confusing in the future. I might be missing the point, but why not change the logic and the name of the scripts that are already there? Or change the names of these scripts to something more descriptive or different from the current scripts. `isis3Startup` vs `isisStartup` doesn't help differentiate between them or why there are two separate ones.

File: isis/scripts/isisStartup.csh (hunk 1 above)
Comment (jessemapel) [reply isis/scripts/isisStartup.csh]: Yeah, I would just return an error message/usage.

**Comment by jessemapel** [comment]
Could we just axe the isis3Startup scripts?

**Comment by scsides** [comment]
@kaitlyndlee, Agreed, but didn't want to blast the old scripts incase someone uses them, like the initIsis.sh script

**Comment by jessemapel** [comment]
Could we add a deprecation warning and give people 6 months/2 releases notice on those scripts going away? I think this is going to be one of the semantic versioning shifts.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**